references/optical_flow/train.py

```diff
@@ -60,16 +60,21 @@ def get_train_dataset(stage, dataset_root):
 
 
 @torch.no_grad()
-def _validate(model, args, val_dataset, *, padder_mode, num_flow_updates=None, batch_size=None, header=None):
+def _evaluate(model, args, val_dataset, *, padder_mode, num_flow_updates=None, batch_size=None, header=None):
     """Helper function to compute various metrics (epe, etc.) for a model on a given dataset.
 
     We process as many samples as possible with ddp, and process the rest on a single worker.
     """
     batch_size = batch_size or args.batch_size
+    device = torch.device(args.device)
 
     model.eval()
 
-    sampler = torch.utils.data.distributed.DistributedSampler(val_dataset, shuffle=False, drop_last=True)
+    if args.distributed:
+        sampler = torch.utils.data.distributed.DistributedSampler(val_dataset, shuffle=False, drop_last=True)
+    else:
+        sampler = torch.utils.data.SequentialSampler(val_dataset, drop_last=True)
```

Comment:
This will raise an error because  `SequentialSampler` doesn't accepts a `drop_last` parameter.

```suggestion
        sampler = torch.utils.data.SequentialSampler(val_dataset)
```

Reply:
Ah yes, somehow I can complete the training without getting error before... But I will fix this, thanks!